src/Discord.Net.Commands/Extensions/MessageExtensions.cs

```diff
@@ -19,7 +19,7 @@ public static class MessageExtensions
         public static bool HasCharPrefix(this IUserMessage msg, char c, ref int argPos)
         {
             var text = msg.Content;
-            if (text.Length > 0 && text[0] == c)
+            if (!string.IsNullOrEmpty(text) && text.Length > 0 && text[0] == c)
```

Comment:
`string.IsNullOrEmpty` makes the `text.Length > 0` check redundant

Reply:
Removed length check.